packages/babel-preset-expo/index.js

```diff
@@ -45,7 +45,7 @@ module.exports = function (api, options = {}) {
       {
         // Defaults to `automatic`, pass in `classic` to disable auto JSX transformations.
         runtime: (options && options.jsxRuntime) || 'automatic',
-        ...(options?.jsxRuntime !== 'classic' && {
+        ...(options && options.jsxRuntime && options.jsxRuntime !== 'classic' && {
```

Comment:
I think this is the correct translation, no?
```suggestion
        ...(options && options.jsxRuntime !== 'classic' && {
```

Reply:
Yep you're right